Adyen/Formatters/Formatter.swift

```diff
@@ -7,18 +7,20 @@
 import Foundation
 
 /// Formats a value for display purposes.
-public protocol Formatter {
+public protocol Formatter: Sanitizer {
     
     /// Formats the given value.
     ///
     /// - Parameter value: The value to format.
     /// - Returns: A formatted version of the given value.
     func formattedValue(for value: String) -> String
     
+}
+
+public protocol Sanitizer {
```

Comment:
perhaps we can come up with a better name for this, because `sanitizing` means something else in this context, removing illegal characters and/or padding.

Reply:
Any suggestion here? Cause this is the term we're using all across the code

Reply:
I think in some cases we are using the term correctly, if you check `NumericFormatter.swift` line 16, we are actually calling `sanitizeValue` from `formattedValue`. But unformatting is something else. Maybe `Unformatter` or `Restorer`?

Reply:
sanitizing means indeed removing illegal characters, and that's what it does, right?.

Reply:
actually the comment on lines 21-24 is wrong, but the name of the protocol is good.

Reply:
Updated the documentation to fix the confusion